.github/workflows/unit-and-integration-tests-unified.yml

```diff
@@ -35,6 +36,7 @@ on:
       - 'examples/**/jupyter-notebooks/**'
       - 'web-console/**'
       - 'website/**'
+      - 'distribution/bin/**'
```

Comment:
@kfaraz re:
> I still haven't figured out how to ensure that UT workflows are not triggered when changes are made to distribution/bin files. Need to handle that too.

This `paths-ignore` config you have added should skip the UT/IT workflows. Once the change is merged into master, you can test it by modifying one of the Python scripts (or in a private branch). Since there are changes to the workflow files in this PR, all the jobs will be triggered.

Reply:
Ah, I feared it was that but assumed that workflow files would be excluded by default.